Changelog.md

```diff
@@ -19,6 +19,7 @@ Bugfixes:
  * SMTChecker: Fix formatting of unary minus expressions in invariants.
  * SMTChecker: Fix internal compiler error when reporting proved targets for BMC engine.
  * TypeChecker: Fix segfault when assigning nested tuple to tuple.
+ * Yul AST: Fix ``nativeSrc`` attributes in optimized IR AST referring to locations in unoptimized IR.
```

Comment:
Whilst definitely a fix, it's not *technically* one since it isn't documented anywhere. Should we perhaps not refer to it as a fix?

Reply:
I don't think we ever limited the bugfix section to documented behavior. If it's visible to the user and was not intentional, it counts as a bugfix in my opinion.

We usually barely provide any detail for bugfixes anyway, so I think it's often hard for users to determine how the behavior changed, regardless of whether it was documented or not.